pkg/storage/mto_milestone.go

```diff
@@ -12,6 +12,20 @@ import (
 	"github.com/cms-enterprise/mint-app/pkg/sqlutils"
 )
 
+// MTOMilestoneGetByIDsLoader returns all milestones by the provided ids
+func MTOMilestoneGetByIDsLoader(np sqlutils.NamedPreparer, _ *zap.Logger, ids []uuid.UUID) ([]*models.MTOMilestone, error) {
```

Comment:
The other methods/loaders in this file reference the singular when referring to the keys, should we have this mimic that? (also lines up with Storage/SQL?)


```suggestion
func MTOMilestoneGetByIDLoader(np sqlutils.NamedPreparer, _ *zap.Logger, ids []uuid.UUID) ([]*models.MTOMilestone, error) {
```

Reply:
good call! Thanks